README.md

```diff
@@ -65,6 +65,14 @@ Test the application on url http://localhost:8080/
 You can also use sbt instead of gradle. It offers a better integration with intelliJ and allows debug.
 
 
+## Using local repositories
+If you do not want to clone the repositories with git you can also specify the local pathes of the repositories :
```

Comment:
hello, there is a small typo at `pathes` -> `paths`

Reply:
done